api-extractor/src/definitions/ApiDocumentation.ts

```diff
@@ -165,6 +157,11 @@ export default class ApiDocumentation {
   public apiTag: ApiTag;
 
   /**
+   * The number of times an "API Tag" was encountered in the JSDoc.
+   */
+  public apiTagCount: number;
```

Comment:
ApiDocumentation's job is to ensure the syntax is correct.  External code shouldn't really care this count is >1.  If it ==0 then the value will be Unknown.

So we should not expose apiTag.